docs/conversion/sakai_21_0-21_1_oracle_conversion.sql

```diff
@@ -1,11 +1,11 @@
 -- START SAK-45137
 UPDATE rbc_rating SET title = 'Title' WHERE title IS NULL;
 ALTER TABLE rbc_rating MODIFY title VARCHAR2(255) NOT NULL;
-ALTER TABLE rbc_rating ALTER title DROP DEFAULT;
+ALTER TABLE rbc_rating ALTER title DEFAULT NULL;
```

Comment:
@vacercon I'm not sure about this change, the previous alter sentence sets title as not null.

Reply:
1)I don't really understand what you want to do since the mysql script updates the values differently from Oracle. 
Mysql sentence is UPDATE rbc_rating SET title = '' WHERE title IS NULL;
Oracle sentence is UPDATE rbc_rating SET title = 'Title' WHERE title IS NULL;

2)ALTER TABLE rbc_rating ALTER title DROP DEFAULT;
Is not a valid Oracle sentence.
Which is the intention?

In the sentence
ALTER TABLE rbc_rating MODIFY title VARCHAR2(255) NOT NULL;
it is decided that the value is not null
but in this one
ALTER TABLE rbc_rating ALTER title DEFAULT NULL;
its decided that the DEFAULT value is.
This is not a contradiction although it may seem so.

Reply:
I have recently had to deal with this issue of half the Oracle scripts being MySQL. In this case I just removed the ALTER TABLE rbc_rating ALTER title DROP DEFAULT; statement because in our database that column did not have a default value anyway. Note also that not having a default and having a default of NULL are different things in Oracle. It is actually not possible to drop a default in Oracle if one is set. You have to recreate the entire column and copy over the data from the original column.

Reply:
That's why I asked, maybe just remove the sentence is fine.

Reply:
ALTER TABLE rbc_rating ALTER title DROP DEFAULT; -- ERROR DESA, TEST Y PROD
We do not execute this directly.